setup.py

```diff
@@ -57,6 +57,7 @@
     'mock>=2.0.0',
     'parameterized>=0.6.1',
     'simple-crypt>=4.1.7',
+    'ruamel.yaml'
```

Comment:
nit: alpha-sort `ruamel.yaml` in this list please

Reply:
This is actually only needed in the installer, will go ahead and remove it.